.github/workflows/podman-desktop-e2e-windows.yaml

```diff
@@ -0,0 +1,158 @@
+      fork:
+        default: 'containers'
+        description: 'Podman Desktop repo fork'
+        type: string
+        required: true
+      branch:
+        default: 'main'
+        description: 'Podman Desktop repo branch'
+        type: string
+        required: true
+
+jobs:
+  windows:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        windows-version: ['10']
+        windows-featurepack: ['22h2-ent']
+
+    steps:
+    - name: Create instance
+      run: |
+        # Create instance
+        podman run -d --name windows-create --rm \
+          -v ${PWD}:/workspace:z \
+          -e ARM_TENANT_ID=${{ secrets.ARM_TENANT_ID }} \
+          -e ARM_SUBSCRIPTION_ID=${{ secrets.ARM_SUBSCRIPTION_ID }} \
+          -e ARM_CLIENT_ID=${{ secrets.ARM_CLIENT_ID }} \
+          -e ARM_CLIENT_SECRET='${{ secrets.ARM_CLIENT_SECRET }}' \
+          quay.io/rhqp/qenvs:v0.0.5 azure \
+            windows create \
+            --project-name 'windows-desktop' \
+            --backed-url 'file:///workspace' \
+            --conn-details-output '/workspace' \
+            --windows-version '${{ matrix.windows-version }}' \
+            --windows-featurepack '${{ matrix.windows-featurepack }}' \
+            --vmsize 'Standard_D8a_v4' \
+            --tags project=podman-desktop \
+            --spot
+        # Check logs 
+        podman logs -f windows-create
+
+    - name: Check instance system info
+      run: |
+        ssh -i id_rsa \
+          -o StrictHostKeyChecking=no \
+          -o UserKnownHostsFile=/dev/null \
+          -o ServerAliveInterval=30 \
+          -o ServerAliveCountMax=1200 \
+          $(cat username)@$(cat host) "systeminfo"
+
+    - name: Emulate X session 
+      run: |
+        # use fake rdp to emulate an active x session
+        podman run -d --name x-session \
+          -e RDP_HOST=$(cat host) \
+          -e RDP_USER=$(cat username) \
+          -e RDP_PASSWORD=$(cat userpassword) \
+          quay.io/rhqp/frdp:v0.0.1
+        # Wait until the x session has been created
+        podman wait --condition running x-session
+        # Check logs for the x session
+        podman logs x-session
+
+    - name: Run PDE2E builder and setup image
+      run: |
+        podman run -d --name pde2e-builder-run \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-builder:v0.0.1-snapshot  \
+              pd-e2e/run.ps1 \
+                  -targetFolder pd-e2e \
+                  -resultsFolder results \
+                  -fork ${{ inputs.fork }} \
+                  -branch ${{ inputs.branch }}
+        # check logs
+        podman logs -f pde2e-builder-run
+        
+    - name: Run podman desktop e2e
```

Comment:
Can you add some comment explaining why we keep using this step here (maybe link to the issue on podman-desktop??) so we can ensure if it is solved we should remove the step from here

Reply:
#19 should explain this. We need to install podman via podman desktop using GOAX in order to be able to run podman desktop e2e playwright tests (pulling images, starting containers, etc.).
I do not think we will remove this step at all.

Reply:
If that step is not gonna be removed ever we can use this step as the one for running those test as so have only one flow, the other option is adding the podman machine steps to your runner WDYT?